podcasts/End of Year/Stories/ListeningTimeStory.swift

```diff
@@ -13,37 +13,48 @@ struct ListeningTimeStory: ShareableStory {
 
     var body: some View {
         GeometryReader { geometry in
-            VStack(spacing: 0) {
-                StoryLabelContainer(topPadding: geometry.size.height * Constants.topPadding, geometry: geometry) {
-                    let time = listeningTime.storyTimeDescription
-                    if NSLocale.isCurrentLanguageEnglish {
-                        StoryLabel(L10n.eoyStoryListenedToUpdated("\n\(time)\n"), highlighting: [time], for: .title)
-                    } else {
-                        StoryLabel(L10n.eoyStoryListenedTo("\n\(time)\n"), highlighting: [time], for: .title)
-                    }
-                    StoryLabel(FunMessage.timeSecsToFunnyText(listeningTime), for: .subtitle)
-                        .opacity(0.8)
+            PodcastCoverContainer(geometry: geometry) {
+                StoryLabelContainer(geometry: geometry) {
+                    StoryLabel(L10n.eoyStoryListenedToTitle, for: .title, geometry: geometry)
+                    StoryLabel(L10n.eoyStoryListenedToSubtitle, for: .subtitle, color: Color(hex: "8F97A4"), geometry: geometry)
                 }
 
-                // Podcast images angled to fill the width of the view
-                let size = 0.30 * geometry.size.height
-
-                HStack(spacing: 20) {
-                    ForEach(Constants.displayedPodcasts, id: \.self) {
-                        podcastCover($0, size: size)
-                    }
+                let fontSize = Int(listeningTime.firstNumber) ?? 0 <= 21 ? 0.4 : 0.3
+                ContentSizeGeometryReader { listeningTimeReader in
+                    Text(listeningTime.firstNumber)
+                        .font(.custom("DM Sans", size: geometry.size.height * fontSize))
+                        .fontWeight(.regular)
```

Comment:
In the original designs, the font has a -10% letter spacing. I wasn't able to make `kerning` or `tracking` to work. The text gets clipped. Any help is very much appreciated.

Reply:
We could manually split the number into an HStack and then apply a negative spacing to it. But this causes its own problems. 

I think for now, this is fine.